src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Same as above. @rainersigwald, what was the resolution for the state file? does it exist / work on .net core? I remember @danmosemsft did something about it.